src/main/java/seedu/address/logic/commands/ModeCommand.java

```diff
@@ -47,7 +54,12 @@ public ModeCommand(String newGameModeName, String newGameDifficultyName) {
     public CommandResult executePrimitive(Model model, CommandHistory history) {
         requireNonNull(model);
 
-        if (newGameDifficultyName == null) {
+        if (newGameModeName == null && newGameDifficultyName == null) {
+            GameMode gameMode = model.getTaskManager().getGameManager().getGameMode();
+            String modeName = gameMode.getClass().getSimpleName();
+            String shortName = modeName.split("Mode")[0];
```

Comment:
Might want to write a parser for this?

Reply:
Rewritten as a method within GameMode to return the name.